### PR TITLE
Awards Screen - Change bars from labels to lines

### PR DESCRIPTION
**Pull request by CrazyEddieTK**
. Fixes part of #2147

### CODE REVIEW

**Comment by sprunk** [comment]
Does this close #2147 (seeing as jitter is also fixed)?

File: LuaUI/Widgets/gui_chili_endgamewindow.lua [hunk 1]
```diff
@@ -166,7 +167,8 @@ local function SetupAwardsPanel()
 				awardSubPanel:AddChild( MakeAwardPanel(awardType, record) )
 			end
 			
-			Label:New{ caption = string.rep('-', 300), textColor = {0.4,0.4,0.4,0.4}; autosize=false; width='100%'; height=5; parent=awardSubPanel } --spacer label to force a "line break"
+--			Label:New{ caption = string.rep('-', 300), textColor = {0.4,0.4,0.4,0.4}; autosize=false; width='100%'; height=5; parent=awardSubPanel } --spacer label to force a "line break"
```
Comment (sprunk) [inline LuaUI/Widgets/gui_chili_endgamewindow.lua]: Remove the 300 dashes line altogether.

Comment (CrazyEddieTK) [inline LuaUI/Widgets/gui_chili_endgamewindow.lua]: Ugh. Forgot. Done now.

**Comment by CrazyEddieTK** [comment]
It fixes the part that _isn't_ the jitter part.

**Pull request merged**